lib/leafblower_web/controllers/game_splash_live.ex

```diff
@@ -68,7 +68,7 @@ defmodule LeafblowerWeb.GameSplashLive do
   def handle_event("join_by_code", %{"code" => params}, socket) do
      {:noreply,
      socket
-     |> push_redirect(to: Routes.live_path(socket, LeafblowerWeb.GameLive, params["code"]), replace: true)}
+     |> push_redirect(to: Routes.live_path(socket, LeafblowerWeb.GameLive, String.upcase(params["code"])), replace: true)}
```

Comment:
I'm thinking we can move this in the changeset
```diff
@@ -137,6 +138,7 @@ defmodule LeafblowerWeb.GameSplashLive do
   defp cast_game_code(params \\ %{}) do
     {%{}, %{code: :string}}
     |> Ecto.Changeset.cast(params, [:code])
+    |> Ecto.Changeset.update_change(:code, &String.upcase/1)
     |> Ecto.Changeset.validate_required([:code])
     |> Ecto.Changeset.validate_length(:code, max: 5)
   end
```

Reply:
Nice! Much cleaner